src/docs/product/profiling/index.mdx

```diff
@@ -17,6 +17,7 @@ Profiling is currently in beta. Beta features are still in-progress and may have
   * Node.js
   * Python
   * PHP (including Laravel and Symfony)
+  * Ruby
```

Comment:
I think we should add the `[experimental]` label (like the Rust one below this) until we're ready to GA